old/CommandLineCommand.java

```diff
@@ -0,0 +1,7 @@
+package jlifx.commandline;
+
+import java.io.PrintStream;
+
+public interface CommandLineCommand {
+	boolean execute(String[] args, PrintStream out) throws Exception;
```

Comment:
\[Checkstyle\] ERROR: File contains tab characters \(this is the first instance\)\.